site/docs/contribute.md

```diff
@@ -45,6 +45,17 @@ The Iceberg community prefers to receive contributions as [Github pull requests]
 * If a PR is related to an issue, adding `Closes #1234` in the PR description will automatically close the issue and helps keep the project clean
 * If a PR is posted for visibility and isn't necessarily ready for review or merging, be sure to convert the PR to a draft
 
+### Merging Pull Requests
+
+Most pull requests can be merged once a committer is satisfied with the code in the PR. For larger changes or additions to public
+APIs committers will wait at least 24 hours before merging to ensure there is no additional feedback from members of the community. As in
+all ASF governed projects committers are expected to act in the best interest of the project. If a committer feels there might be a conflict
+of interest with a pull request they review, they are encouraged to ask for another committer to look at the pull request.
```

Comment:
Could we define what "a conflict of interest" means, or at least give any examples?

Reply:
Does this mean cases like a committer working on pyiceberg go to merge something directly in iceberg-spark? Just to given an arbitrary example.

Reply:
I left this intentionally vague.  In general committers are expected to use there best judgement. If there are explicit examples that have happened in the community where it was believe something should not been committed we can include those.  Otherwise, I think this is place that can be refined if the PMC thinks some changes are being merged prematurely.

Reply:
This goes to the ethos of community over code and trust in committers.